draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -311,118 +435,80 @@ and media access delays and link-layer retransmission lead to increased jitter
-{{?RFC3714}}, or should include some notion of "full backoff".  For
-"full backoff", at some point the algorithm would reduce the
-sending rate to one packet per round-trip time and then
-exponentially backoff the time between single packet
-transmissions if congestion persists.  Exactly when either "full
-backoff" or a pause in sending comes into play will be
-algorithm-specific.  However, as discussed in {{!RFC2914}}, this
-requirement is crucial to protect the network in times of extreme
-congestion.
+(TODO: Write this section)
 
-: If "full backoff" is used, this bullet does not require that the
-full backoff mechanism must be identical to that of TCP
-{{?RFC2988}}.  As an example, this bullet does not preclude full
-backoff mechanisms that would give flows with different round-
-trip times comparable caapcity during backoff.
+## Satellite
 
-(6)
-: Protection Against Bufferbloat
+Satellite links often have long delays {{?RFC2488}} and high delay bandwidth
+products{{?RFC3649}}.
 
-: The alternate congestion control mechanism should reduce its sending
-rate if the round trip time (RTT) significantly increases. Exactly how
-the algorithm reduces its sending rate is algorithm specific.
+## Misbehaving Nodes
 
-: Bufferbloat {{Bufferbloat}} refers to the building of long queues in
-the network. Many network routers are configured with very large buffers.
-If congestion starts happening, classic TCP congestion control algorithms
-{{!RFC5681}} will continue sending at a high rate until the buffer fills
-up completely and packet losses occur. Every connection going through
-that bottleneck will experience high latency.
-This is particularly bad for highly interactive applications like games,
-but it also affects routine web browsing and video playing.
+The proposal should explore how the alternate congestion control
+mechanism performs with misbehaving senders, receivers, or
+routers.  In addition, the proposal should explore how the
+alternate congestion control mechanism performs with outside
+attackers.  This can be particularly important for congestion
+control mechanisms that involve explicit feedback from routers
+along the path.
 
-: This problem became apparent in the last decade and was not discussed in
-the Congestion Control Principles published in September 2002 {{!RFC2914}}.
-The classic congestion control algorithm {{!RFC5681}} and the widely deployed
-Cubic algorithm {{?RFC9438}} do not address it, but newly designed congestion
-control algorithms have the opportunity to improve the state of the art.
+As an example from an Experimental RFC, performance with
+misbehaving nodes and outside attackers is discussed in Sections
+9.4, 9.5, and 9.6 of {{?RFC4782}} (Quick-Start).  This includes
+discussion of misbehaving senders and receivers; collusion
+between misbehaving routers; misbehaving middleboxes; and the
+potential use of Quick-Start to attack routers or to tie up
+available Quick-Start bandwidth.
+(TODO: Write this sectoin)
```

Comment:
/section/ ... and can we add to the note, that this is *when* the alternative CC relies on the presence or operation of a mechanism implemented in a router.

Reply:
I deleted thte TODO, because it's an artifact of the edit.

The explicit signal discussion is in the first paragraph of this section. I don't see a need to repeat it.